vra7/resource_vra7_deployment.go

```diff
@@ -94,8 +96,24 @@ func resourceVra7Deployment() *schema.Resource {
 			"resource_configuration": resourceConfigurationSchema(true),
 			"lease_days": {
 				Type:     schema.TypeInt,
-				Computed: true,
 				Optional: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					return false
+				},
+			},
+			"lease_end": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					value := v.(string)
+					matched, _ := regexp.MatchString(`^([0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1]))$|^([0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1]) (2[0-3]|[01][0-9]):[0-5][0-9])$`, value)
```

Comment:
I'm worried about this regexp. This string needs to be broken out in a top level variable and then a lot of unit tests with positive and negative tests in order to validate it. Or, use the time Parse function to validate.